packages/forms/lib/src/range-slider/components/range-slider/range-slider.component.ts

```diff
@@ -88,9 +89,84 @@ export class RangeSliderComponent implements OnInit, ControlValueAccessor {
 			return;
 		}
 
+		this.hasFocus = true;
 		this.active = handle;
 	}
 
+	public toggleFocus(hasFocus, element, $event) {
+		if (this.isDisabled) {
+			return;
+		}
+
+		$event.preventDefault();
+		this.hasFocus = hasFocus;
+		this.active = element;
+	}
+
+	@HostListener('document:keydown', ['$event'])
+	public onKeyDown($event) {
+		if (!this.hasFocus) {
+			return;
+		}
+
+		const key = $event.keyCode;
+
+		if (!~[35, 36, 37, 38, 39, 40].indexOf(key)) {  // tslint:disable-line:no-bitwise
+			return;
+		}
+		// 35: end
```

Comment:
Suggestion: 
You can write this as 'self-documenting code' by putting the keycodes in a const and using them. This way the comments don't have to be maintained if something changes
```javascript
const keycode = {
  left: 37,
  right: 39
};
if (!~[keycode.left, keycode.right].indexOf(key)){}
```

Reply:
Fixed ;-)